circuits/circom/verify_signature.circom

```diff
@@ -118,11 +118,13 @@ template VerifySignature(k) {
 
   signal output valid;
   
-  component M = Hasher(k);
-  M.key <== 0;
+  component M = Hasher11();
   for (var i = 0; i < k; i++){
     M.in[i] <== preimage[i];
   }
+  for (var i = k; i < 11; i++){
```

Comment:
I think we should do another PR to set a constant number of inputs for `VerifySignature`: see #99